heltour/tournament/tasks.py

```diff
@@ -317,31 +320,41 @@ def update_slack_users():
             p.save()
 
 
+def _expire_bad_tokens(*, league_games, bad_token):
+    # set expiration of rejected token to yesterday, so we know to not use it anymore.
+    for game in league_games:
+        if game.get_white_access_token()==bad_token:
+            game.white.oauth_token.expires = timezone.now() + timedelta(days=-1)
+            game.white.oauth_token.save()
+            return None # only one oauth_token can be the bad token, so we do not need to proceed the function
+        if game.get_black_access_token()==bad_token:
+            game.black.oauth_token.expires = timezone.now() + timedelta(days=-1)
+            game.black.oauth_token.save()
+            return None
```

Comment:
I want to do something like the below (which would require some more or different getter methods):

```suggestion
        for token in [game.get_white_oauth_token(), game.get_black_oauth_token()]:
        if oauth_token.access_token==bad_token:.
            oauth_token.expires = timezone.now() + timedelta(days=-1)
            oauth_token.save()
            break # only one oauth_token can be the bad token, so we do not need to proceed the function
```